src/plugins/intel_cpu/src/cpu_memory.cpp

```diff
@@ -212,7 +212,7 @@ void MemoryMngrWithReuse::setExtBuff(void *ptr, size_t size) {
 bool MemoryMngrWithReuse::resize(size_t size) {
     constexpr int cacheLineSize = 64;
     bool sizeChanged = false;
-    if (size > m_memUpperBound) {
+    if (size > m_memUpperBound || m_data.get() == nullptr) { // conform to ov::Tensor which always has an addr even for empty tensor
```

Comment:
Unluckily both them would return a non-null ptr when the size is 0.

We'd better let ITensor::data() return the same pointer of MemoryManager::resize(). Otherwise we need extra check to make sure this tensor and memory sharing the same address.

Reply:
updated, we won't care if it return nullptr or not in case of size 0.